### PR TITLE
AQ-47_setup_faker

### PR DESCRIPTION
**Pull request by olivergav**
# Description

Added Faker.js and made component for test work of it.

@faker-js/faker@7.6.0

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

It has been not

# Checklist:

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] The related memme has been added

image

### CODE REVIEW

File: quiz/web/tsconfig.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "lib": ["DOM", "DOM.Iterable", "ESNext"],
         "allowJs": false,
         "skipLibCheck": true,
-        "esModuleInterop": false,
+        "esModuleInterop": true,
```
Comment (olivergav) [inline quiz/web/tsconfig.json:8]: https://fakerjs.dev/guide/usage.html#typescript-support
https://www.youtube.com/watch?v=aKMZq7bk-D8

File: quiz/web/src/modules/genericTests/FakerTest.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { faker } from "@faker-js/faker";
+import type { SexType } from "@faker-js/faker";
+
+type SubscriptionTier = "free" | "basic" | "pro";
+
+class User {
```
Comment (pawelkonior) [inline quiz/web/src/modules/genericTests/FakerTest.tsx:7]: W TS to jest niepotrzebne, można użyć private albo protected, kod się skróci 3x, a będzie robił to samo. https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd

**Review by pawelkonior — CHANGES_REQUESTED** [review]
Popraw ten konstruktor do fakera, reszta wygląda spoczko.

File: quiz/web/src/modules/genericTests/FakerTest.tsx (hunk 2 above)
Comment (olivergav) [reply quiz/web/src/modules/genericTests/FakerTest.tsx:7]: Poprawione, z użyciem public, ponieważ na private lub protected sypało błędami TS

**Review by pawelkonior — APPROVED** [review]
Good job

**Review by ToJestKrzysio — APPROVED** [review]

**Pull request merged**